src/mono/mono/metadata/metadata.c

```diff
@@ -5103,13 +5103,17 @@ mono_metadata_custom_attrs_from_index (MonoImage *meta, guint32 index)
 	/* FIXME: Index translation */
 
 	gboolean found = tdef->base && mono_binary_search (&loc, tdef->base, table_info_get_rows (tdef), tdef->row_size, table_locator) != NULL;
-	if (!found && !meta->has_updates)
-		return 0;
-
-	if (G_UNLIKELY (meta->has_updates)) {
-		if (!found && !mono_metadata_update_metadata_linear_search (meta, tdef, &loc, table_locator))
+	if (!found)
```

Comment:
```suggestion
	if (!found) {
```

Reply:
It's not technically required by C, but  it's worth adding it anyway to improve clarity, IMO.

Reply:
Added.